readme.md

```diff
@@ -157,7 +157,7 @@ console.log(chalk.styles.red.open + 'Hello' + chalk.styles.red.close);
 - `red`
 - `green`
 - `yellow`
-- `blue` *(on Windows the bright version is used as normal blue is illegible)*
+- `blue` *(on Windows the bright version is used, as normal blue is illegible)*
```

Comment:
Thanks for the review. Not a native English speaker. It didn't make sense to me without a punctuation mark.